shared/common-adapters/avatar.js

```diff
@@ -132,10 +133,27 @@ function _followIconSize(size: number, followsYou: boolean, following: boolean)
   return 0
 }
 
+let _askQueue = {}
+let _askDispatch = null
+// We queue up the actions across all instances of Avatars so we don't flood the system with tons of actions
+const _askForUserDataQueueUp = (username: string, dispatch: Dispatch) => {
```

Comment:
was hoping to avoid this but its just too problematic (especially since the actions go across the ipc bridge). we now have a debounced local function to get avatars